### PR TITLE
Multiple microservices demo

### PR DESCRIPTION
**Pull request by wujimin**

### CODE REVIEW

**Comment by wujimin** [comment]
1.allow multiple microservices in one process
2.unit test speed optimized

File: core/src/main/java/io/servicecomb/core/Const.java [hunk 1]
```diff
@@ -34,5 +34,5 @@ private Const() {
 
     public static final String SRC_MICROSERVICE = "x-cse-src-microservice";
 
-    public static final String DEST_MICROSERVICE = "x-cse-dest-microservice";
+    public static final String TARGET_MICROSERVICE = "x-cse-target-microservice";
```
Comment (WillemJiang) [inline core/src/main/java/io/servicecomb/core/Const.java]: Is this header used by other outside service?

Comment (wujimin) [inline core/src/main/java/io/servicecomb/core/Const.java]: when target process have multiple microserives,  and consumer do not use cse sdk
then consumer must set x-cse-target-microservice in header

Comment (WillemJiang) [inline core/src/main/java/io/servicecomb/core/Const.java]: If the header is used internally, it's OK to change the header name, but if it is used from outside, we should keep the old header there.

Comment (wujimin) [inline core/src/main/java/io/servicecomb/core/Const.java]: no problem
this is the first time to use it.

**Pull request merged**